arcaflow_plugin_fio/fio_plugin.py

```diff
@@ -17,6 +17,23 @@
 )
 
 
+def split_json_and_errors(fio_output: str):
+    # Sometimes fio informational messages are included at the top of the JSON output.
+    # Separate the json data from the errors and return them as a tuple.
```

Comment:
I suggest including in this text references to the (currently) open fio Issue as well as a comment and/or pointer to the fio test that you cribbed this from.